fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DeleteResourcePersister.java

```diff
@@ -46,13 +48,14 @@ public void persist(final OCFLPersistentStorageSession session, final ResourceOp
         final var fedoraResourceRoot = mapping.getRootObjectIdentifier();
         final var resourceId = operation.getResourceId();
         final var objectSession = session.findOrCreateSession(mapping.getOcflObjectId());
+        final var isRdf = ((DeleteResourceOperation) operation).isRdf();
```

Comment:
Would it be an option to call `readHeaders` here in the persister to determine that the object being deleted is a binary or not, rather than passing it down through the Operation stuff?

Reply:
Maybe...the OCFL layer is still a bit of a tangle moving stuff around too me. Is that preferable to passing it down?

Reply:
Ugh, ok I'm going to start over with a fresh branch instead of trying to revert all the changes here. I'll force push over top later.

Reply:
I don't have a _strong_ opinion on it,  but it does seem cleaner not to pass the isRdf flag.

Reply:
Sorry to blow things up on you like that. Just seems preferable to have the data coming from as close as possible if its an option, fewer things to lineup or places for things to go wrong.